src/SDKs/DataMigration/DataMigration.Tests/ScenarioTests/CRUDProjectTests.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using DataMigration.Tests.Helpers;
+using Microsoft.Azure.Management.DataMigration;
+using Microsoft.Azure.Management.DataMigration.Models;
+using Microsoft.Azure.Management.Resources;
+using Microsoft.Rest.ClientRuntime.Azure.TestFramework;
+using System.Net;
+using Xunit;
+
+namespace DataMigration.Tests.ScenarioTests
+{
+    public class CRUDProjectTests : CRUDDMSTestsBase
+    {
+        [Fact]
```

Comment:
@hitenjava use the latest resource manager 1.6.0 at least to record all your tests.

Reply:
@shahabhijeet - You mean Microsoft.Azure.ResourceManager? I only see v1.1.0-preview as the latest.

Reply:
@shahabhijeet Where can we find the resource manager 1.6.0 to reference?

Reply:
Had a call with Abhijeet, link to resource manager - https://www.nuget.org/packages/Microsoft.Azure.Management.ResourceManager/1.6.0-preview

I will update tests to use this.